src/buildPresets/buildPresets.ts

```diff
@@ -12,7 +12,7 @@ export const buildPresets: IBuildPreset[] = [
         displayName: 'Angular',
         appLocation: '/',
         apiLocation: 'api',
-        outputLocation: 'dist'
+        outputLocation: 'dist/<Project name>'
```

Comment:
nit: Could we do:
```suggestion
        outputLocation: 'dist/<project-name>'
```

to match Angular's website: https://angular.io/guide/deployment
> The output folder is dist/project-name/ by default.

Reply:
Good suggestion 👍

Reply:
Can we modify `validateInput` for the output step to basically check for this placeholder and make sure they change it? Also it would be nifty if just the placeholder is selected when you open the input box, but not sure if that's possible.

Reply:
Did some research and we're only able to specify what part of the value is selected when using the native `window.showInputBox`, instead of creating the `InputBox` manually. 

I was able to implement the validate input idea :+1: